### PR TITLE
Fixes #208:Scrolling issue in TripIndicatorPackingDialog in Landscape mode

### LINKED ISSUE #208: Scrolling issue in TripIndicatorPackingDialog in Landscape mode
**Current Behaviour**
 ListView for the **Packing Items**  dialog  is not scrollable in **Landscape** mode.

ezgif com-optimize 5

**Expected Behaviour**
List for the packing items in TripIndicatorPackingActivity must be scrollable even in the landscape mode.

**Fix**
We can add a touch listener to Parent's ScrollView  that disables the parent's touch intercept when a touch starts in the child(ListView in our case), and then enables it back when the touch starts in the parent  again,
so that separate scrolling can occur easily in the parent's view as well as in the child view.

@chhavip @yatna I have a fix for this.Can I send PR for the same? Thanks :)

### PR DESCRIPTION
**Pull request by HimanshuS01**
Fixes issue #208 
 
**GIF for the changes I have made** 
ezgif com-optimize 6

@chhavip @yatna @anubhakushwaha Please review this PR. Thanks :)

### CODE REVIEW

File: malaria-app-android/src/main/java/com/peacecorps/malaria/activities/TripIndicatorPackingActivity.java [hunk 1]
```diff
@@ -233,6 +237,19 @@ public void onClick(View v) {
 
         tripDrugName=whichDrug.getText().toString();
 
+        parentScroll.setOnTouchListener(new View.OnTouchListener() {
+            public boolean onTouch(View v, MotionEvent event) {
+                findViewById(R.id.listV).getParent().requestDisallowInterceptTouchEvent(false);
```
Comment (yatna) [inline malaria-app-android/src/main/java/com/peacecorps/malaria/activities/TripIndicatorPackingActivity.java]: Remove extra line.

File: malaria-app-android/src/main/java/com/peacecorps/malaria/activities/TripIndicatorPackingActivity.java [hunk 2]
```diff
@@ -233,6 +237,19 @@ public void onClick(View v) {
 
         tripDrugName=whichDrug.getText().toString();
 
+        parentScroll.setOnTouchListener(new View.OnTouchListener() {
+            public boolean onTouch(View v, MotionEvent event) {
```
Comment (yatna) [inline malaria-app-android/src/main/java/com/peacecorps/malaria/activities/TripIndicatorPackingActivity.java]: Remove extra line.

File: malaria-app-android/src/main/java/com/peacecorps/malaria/activities/TripIndicatorPackingActivity.java (hunk 1 above)
Comment (yatna) [inline malaria-app-android/src/main/java/com/peacecorps/malaria/activities/TripIndicatorPackingActivity.java]: Remove extra line

**Comment by HimanshuS01** [comment]
@anubhakushwaha @chhavip Please review this PR. 
Thank you :)

**Pull request merged**